### PR TITLE
Adds coming soon

### LINKED ISSUE #332: Placeholder audits
I think this is a good list for now:

1. Manifest's start_url is in cache storage for offline use
1. Service worker makes use of push notifications, if appropriate
1. Tap targets are appropriately sized for touch
1. Payment forms marked up for with [autocomplete] attributes
1. Login forms marked up with [autocomplete] attributes
1. Input fields use appropriate [type] attributes for custom keyboards
1. Content scrolls at 60fps
1. Touch input gets a response in < 150ms 
1. App is interactive without jank after the first meaningful paint

There are, of course, many more, but I think these sit at the top priority-wise.

### PR DESCRIPTION
**Pull request by paullewis**
Fixes #332 

screen shot 2016-05-17 at 3 00 51 pm

### CODE REVIEW

File: src/aggregators/best-practices/index.js [hunk 1]
```diff
@@ -76,6 +76,46 @@ class IsAccessible extends Aggregate {
...
+      comingSoon: true,
+      description: 'Tap targets are appropriately sized for touch',
+      category: 'UX'
+    };
+
+    criteria['payments-autocomplete'] = {
+      value: true,
+      weight: 0,
+      comingSoon: true,
+      description: 'Payment forms marked up for with [autocomplete] attributes',
```
Comment (paulirish) [inline src/aggregators/best-practices/index.js]: "for with" to "with"

Comment (paullewis) [inline src/aggregators/best-practices/index.js]: Done

**Comment by paulirish** [comment]
nailed it.

**Comment by paullewis** [comment]
All I'm sayin':

screen shot 2016-05-17 at 3 44 37 pm

**Comment by paulirish** [comment]
Favorite.

**Pull request merged**